### ISSUE TITLE
Info displaying partially below the  screen in double monitor setup

### ISSUE THREAD

**Issue by danieltomasz** [regolith-linux/regolith-desktop#226]
Regolith 1.3 on Ubuntu 18.04
As like on the attached screen.
![Screenshot from 2020-02-11 10-00-58](https://user-images.githubusercontent.com/7980381/74223122-4db9c300-4cb6-11ea-9080-45ec51c87c0a.png) 

Also  even when clicking on the secondary monitor, the info will appear only on the  primary monitor.

```
➜ lspci -k | grep -EA3 'VGA|3D|Display'

00:02.0 VGA compatible controller: Intel Corporation HD Graphics P530 (rev 06)
	Subsystem: Lenovo HD Graphics P530
	Kernel driver in use: i915
	Kernel modules: i915
--
01:00.0 VGA compatible controller: NVIDIA Corporation GM107GLM [Quadro M2000M] (rev a2)
	Subsystem: Lenovo GM107GLM [Quadro M2000M]
	Kernel driver in use: nvidia
	Kernel modules: nvidiafb, nouveau, nvidia_drm, nvidia

➜ cat /proc/driver/nvidia/version

NVRM version: NVIDIA UNIX x86_64 Kernel Module  430.64  Sun Oct 27 11:26:12 UTC 2019
GCC version:  gcc version 7.4.0 (Ubuntu 7.4.0-1ubuntu1~18.04.1) 

```

**Comment by kgilmer** [comment]
@danieltomasz is the bug consistent or intermittent?  I have 2 monitors setup at home as well and do not see the issue...wondering how I can reproduce.

**Comment by danieltomasz** [comment]
the window is intact but the centering of the window changed - using shift + master(win) + arrow key I  could  move it to the side of the screen, but when i  again try shift + master + ?, it appear again  centered around (i) icon at the bottom of the screen

**Comment by danieltomasz** [comment]
Changing compositor doesnt work
The behaviour is consistent after pluging and unpluging external monitor via vga
heres xprop output
```
WM_STATE(WM_STATE):
                window state: Normal
                icon window: 0x0
I3_FLOATING_WINDOW(CARDINAL) = 1
_NET_WM_DESKTOP(CARDINAL) = 4294967295
_NET_WM_STATE(ATOM) = _NET_WM_STATE_STICKY, _NET_WM_STATE_SKIP_TASKBAR, _NET_WM_STATE_SKIP_PAGER
WM_HINTS(WM_HINTS):
                Client accepts input or input focus: False
                Initial state is Normal State.
                window id # of group leader: 0x2c00001
_GTK_THEME_VARIANT(UTF8_STRING) =
XdndAware(ATOM) = BITMAP
_GTK_APPLICATION_OBJECT_PATH(UTF8_STRING) = "/org/regolith/remontoire"
_GTK_UNIQUE_BUS_NAME(UTF8_STRING) = ":1.9972"
_GTK_APPLICATION_ID(UTF8_STRING) = "org.regolith.remontoire"
_NET_WM_OPAQUE_REGION(CARDINAL) = 7, 0, 380, 7, 0, 7, 394, 1033
_MOTIF_WM_HINTS(_MOTIF_WM_HINTS) = 0x1, 0x21, 0x0, 0x0, 0x0
_NET_WM_WINDOW_TYPE(ATOM) = _NET_WM_WINDOW_TYPE_SPLASH
_NET_WM_SYNC_REQUEST_COUNTER(CARDINAL) = 46137349, 46137350
_NET_WM_USER_TIME(CARDINAL) = 0
_NET_WM_USER_TIME_WINDOW(WINDOW): window id # 0x2c00004
WM_CLIENT_LEADER(WINDOW): window id # 0x2c00001
_NET_WM_PID(CARDINAL) = 6228
WM_LOCALE_NAME(STRING) = "C.UTF-8"
WM_CLIENT_MACHINE(STRING) = "lenovo-p50"
WM_NORMAL_HINTS(WM_SIZE_HINTS):
                program specified minimum size: 394 by 1040
                program specified maximum size: 394 by 1040
                program specified base size: 0 by 0
                window gravity: NorthWest
WM_PROTOCOLS(ATOM): protocols  WM_DELETE_WINDOW, WM_TAKE_FOCUS, _NET_WM_PING, _NET_WM_SYNC_REQUEST
WM_CLASS(STRING) = "remontoire", "Remontoire"
WM_ICON_NAME(STRING) = "remontoire"
_NET_WM_ICON_NAME(UTF8_STRING) = "remontoire"
WM_NAME(STRING) = "remontoire"
_NET_WM_NAME(UTF8_STRING) = "remontoire"


```

**Comment by danieltomasz** [comment]
Dual monitor setup - remontoire position:
```
➜ sleep 3 && xdotool getwindowfocus getwindowgeometry
Window 123731973
  Position: 1284,126 (screen: 0)
  Geometry: 632x1045

```

**Comment by kgilmer** [comment]
This should be fixed in 1.3.1.
